onyo/lib/tests/test_commands_tree.py

```diff
@@ -3,63 +3,79 @@
 from onyo.lib.inventory import Inventory
 from ..commands import onyo_tree
 
-# TODO: test output?
 
-
-@pytest.mark.ui({'yes': True})
-def test_onyo_tree_errors(inventory: Inventory) -> None:
-    r"""`onyo_tree` must raise the correct error in different illegal or impossible calls."""
+def test_onyo_tree_errors(inventory: Inventory,
+                          capsys) -> None:
+    r"""Raise the correct error for illegal or impossible calls."""
     asset_path = inventory.root / "somewhere" / "nested" / "TYPE_MAKER_MODEL.SERIAL"
     dir_path = inventory.root / 'empty'
 
     # no tree for files
     pytest.raises(ValueError,
                   onyo_tree,
                   inventory,
-                  paths=[(str(asset_path), asset_path)])
+                  path=asset_path,
+                  description='file')
```

Comment:
It's a bit of a nitpick, but putting an invalid `description` in a test that's supposed to test a failure on certain `path`s doesn't make a lot of sense. This does effectively not test what it says it tests, b/c the ValueError could be caused by the description rather than the path.

I realize that's not introduced by this commit, but still: Once we touch it  ...

Reply:
Same goes, of course, for the ones below.

Reply:
Ahh, yeah. I see your point. Keep the failure as tight as possible.

I added the description to make it more explicit/self-documenting. But I will leave that to the comments.

I'll update the PR.